testing/run_tests.py

```diff
@@ -308,7 +308,7 @@ def main():
     if ',' in java_filter or '*' in java_filter:
       print('Can only filter JUnit4 tests by single entire class name, eg "io.flutter.SmokeTest". Ignoring filter=' + java_filter)
       java_filter = None
-    RunJavaTests(java_filter)
+    RunJavaTests(java_filter, android_variant)
```

Comment:
I think this is causing the `NameError: global name 'android_variant' is not defined` in CI.

```suggestion
    RunJavaTests(java_filter, args.android_variant)
```